recipes/mss/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = "1.1.0" %}
+
+
+package:
+  name: mss
+  version: {{ version }}
+
+source:
+    git_url: https://bitbucket.org/wxmetvis/mss.git
+    git_rev: 1.1.0
+
+build:
+    number: 0
+    script: python setup.py install --record record.txt
+    entry_points:
+      - mss = msui.mss_pyui:main
+
+requirements:
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl ==7.26
+    - cycler ==0.10.0
+    - execnet ==1.2.0
```

Comment:
I can help you try to investigate those issues. I believe `basemap-1.0.8` is not to blame, but something of its dependencies might be. Can you post the errors you are seeing? (Or point me to any CI log?)

PS: Let's try to get most of the safe pins (pure Python for example) loose and get the other pins consisted with our [recommendation](https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies). Then we can start debugging.